api/src/main/java/org/motechproject/nms/api/web/CallDetailsController.java

```diff
@@ -54,6 +55,9 @@
     @Transactional
     public void saveCallDetails(@PathVariable String serviceName,
                                 @RequestBody CallDetailRecordRequest callDetailRecordRequest) {
+
+        log(String.format("/%s/callDetails", serviceName), LogHelper.nullOrString(callDetailRecordRequest));
```

Comment:
Not a huge deal, but Spring MVC should have an api to get the URI

http://stackoverflow.com/questions/3686808/spring-3-requestmapping-get-path-value